### PR TITLE
added test for /s

### PR DESCRIPTION
**Pull request by sivaramaaa**
/s honor cmd.hit and search.from/to

### CODE REVIEW

File: t/cmd_search [hunk 1]
```diff
@@ -405,6 +405,34 @@ EXPECT='0x00000000
...
+run_test
+
+NAME='search.from/to for /s'
+FILE='../bins/elf/analysis/x86-simple'
+ARGS=
+CMDS='
+aei
+aeim
+aeip
+e search.from = 0x08048060
```
Comment (radare) [inline t/cmd_search]: Not sure if from/to should be honored over search.in. Maybe makes sense but weshould be clear on that

Comment (sivaramaaa) [reply t/cmd_search]: Yeah currently /s dosen't honor search.in , but it would be fixed soon , currently working on it
Is that ok ??

**Comment by radare** [comment]
The aeim should do aei already and aeip expects an argument or current seek (entrypoint?) if so lgtm

**Comment by sivaramaaa** [comment]
Yeah aeim does the job so i removed aei in test now , also i have sent u a PR so that /s honor search.in , u could merge this PR , once my PR is merged in r2 master repo !!

**Pull request merged**